nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ValidateCsv.java

```diff
@@ -408,9 +407,12 @@ public void process(final InputStream in) throws IOException {
                         listReader.read();
                     }
                     while(listReader.read(cellProcs) != null) {}
-                } catch (final IOException | SuperCsvCellProcessorException e) {
+                } catch (final IOException e) {
                     valid.set(false);
                     logger.error("Failed to validate {} against schema due to {}", new Object[]{flowFile}, e);
+                } catch (final SuperCsvCellProcessorException e) {
+                    valid.set(false);
+                    logger.info("Failed to validate {} against schema due to {}; routing to 'invalid'", new Object[]{flowFile}, e);
```

Comment:
This should be debug, not info, this will occur as a part of normal operation and is only really relevant for debugging.